src/trove_classifiers/__init__.py

```diff
@@ -278,6 +278,7 @@
     "License :: OSI Approved :: Apple Public Source License",
     "License :: OSI Approved :: Artistic License",
     "License :: OSI Approved :: Attribution Assurance License",
+    "License :: OSI Approved :: Blue Oak Model License",
```

Comment:
```suggestion
    "License :: OSI Approved :: Blue Oak Model License (BlueOak-1.0.0)",
```

We've started adding the SPDX identifier after these to avoid issues like what we have in #17.